framework/src/application/network/network.ts

```diff
@@ -139,7 +142,28 @@ export class Network {
 		} else {
 			this._secret = Number(secret);
 		}
-
+		const extractNodeInfoParams = (
+			state: State,
+		): liskP2P.p2pTypes.P2PNodeInfo => {
+			const {
+				networkId,
+				protocolVersion,
+				advertiseAddress,
+				os,
+				version,
+				wsPort,
+				...options
+			} = state;
+
+			const nodeInfo = {
+				networkId,
+				networkVersion: protocolVersion,
```

Comment:
lets create another issue to address the change in the `state`

Reply:
created https://github.com/LiskHQ/lisk-sdk/issues/5528